website/src/pages/product/automation.tsx

```diff
@@ -0,0 +1,231 @@
+                            <p>
+                                Keep your library dependencies **and how you use those libraries** up-to-date and
+                                consistent across all of your organization's code, to avoid old bugs or security
+                                problems in old dependencies, and problems arising from inconsistent dependency version
+                                use across your codebase.
+                            </p>
+                        </IconItem>
+                    </div>
+                </div>
+                <div className="row justify-content-lg-center">
+                    <div className="col-lg-4 mb-4">
+                        <IconItem className="position-relative" icon={PlaylistAddCheckIcon} color="brand-orange">
+                            <h4>Deploy new static analysis gradually in the developer workflow</h4>
+                            <p>
+                                Increase adoption of automatic linters and enable progressively stricter rules across
+                                all of your organization's code, so you can continuously improve the quality of all of
+                                your code. Developers will see diagnostics and fixes in their editor and on their code
+                                host to gently nudge them toward adherence, and you can enforce rules after a certain
+                                time period.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-lg-4 mb-4">
+                        <IconItem className="position-relative" icon={BuildIcon} color="blue">
+                            <h4>Consistent build and deploy configuration</h4>
+                            <p>
+                                Keep the build and deployment configurations up-to-date and consistent across all of
+                                your organization's code, so that you can iterate and deploy continuously and reliably
+                                with DevOps self-sufficiency.
+                            </p>
+                        </IconItem>
+                    </div>
+                </div>
+                <h2 id="easily-define-campaigns-track-progress" className="text-center display-4 mb-3 mt-4">
+                    Easily define campaigns and track their progress
+                </h2>
+
+                <div className="row mt-4">
+                    <div className="row justify-content-md-center">
+                        <p className="col-lg-9">
+                            Sourcegraph's automation lets you create campaigns, which are collections of pull requests
+                            and issues across multiple repositories. The following campaign types are built in, and you
+                            can create other ad-hoc campaigns from the UI or use the Sourcegraph extension API for more
```

Comment:
link the text **Sourcegraph extension API** to https://docs.sourcegraph.com/extensions